RecoLocalFastTime/FTLCommonAlgos/plugins/MTDUncalibRecHitAlgo.cc

```diff
@@ -28,43 +32,44 @@ class MTDUncalibRecHitAlgo : public MTDUncalibratedRecHitAlgoBase {
   FTLUncalibratedRecHit makeRecHit(const ETLDataFrame& dataFrame ) const final;
 
  private:  
-  double adcLSB_, toaLSBToNS_, timeError_;
+  double BTL_adcLSB_, BTL_toaLSBToNS_, BTL_timeError_;
+  double ETL_adcLSB_, ETL_toaLSBToNS_, ETL_timeError_;
 };
 
 FTLUncalibratedRecHit 
 MTDUncalibRecHitAlgo::makeRecHit(const BTLDataFrame& dataFrame ) const { 
```

Comment:
A similar simplification could be done here, though there's more information to pass (`sample.data()`, `sample.toa()`, `dataFrame.id()`, `adcLSB`, `toaLSBToNS`, `timeError`).